ods_ci/tests/Resources/RHOSi.resource

```diff
@@ -38,6 +38,8 @@ Resource            Common.robot
 ...                         Login To OCP Using API
 ...                         Generate Thanos Token
 ...                         Set Thanos Credentials Variables
+...                         Generate Minio Random Credentials
```

Comment:
if they're random why do we want to hide them?

Reply:
makese sense. We can remove it